docker-compose-wiremock.yml

```diff
@@ -6,4 +6,6 @@ services:
     ports:
       - '8080:8080'
       - '8443:8443'
-    command: [ '--https-port', '8443' ]
+    command:
+      - --https-port=8443
+      - --verbose
```

Comment:
is this useful? or accidentally left over?

Reply:
it logs more things on the wiremock container, i added it for troubleshooting but not really needed, removed it